frontend/src/app/playback/playback.component.html

```diff
@@ -1,24 +1,17 @@
-<div class="container">
-  <div *ngIf="isLoading; else content" class="loading-message">
-    <p>{{ message }}</p>
+@if (isLoading) {
+  <p class="loading-message">{{ message }}</p>
+} @else if (!videoItem) {
+  <p class="no-video-message">{{ message }}</p>
+} @else {
+  <h2 class="video-title">{{ videoItem.name }}</h2>
+  <div class="video-wrapper">
+    <iframe
+      [src]="sanitizer.bypassSecurityTrustResourceUrl(getVimeoEmbedUrl(videoItem.vimeo_id))"
+      allow="fullscreen"
+      class="vimeo-player">
+    </iframe>
   </div>
-
-  <ng-template #content>
-    <div *ngIf="videoItem; else noVideo">
-      <h2 class="video-title">{{ videoItem.name }}</h2>
-      <div class="video-wrapper">
-        <iframe
-          [src]="sanitizer.bypassSecurityTrustResourceUrl(getVimeoEmbedUrl(videoItem.vimeo_id))"
-          allow="fullscreen"
-          class="vimeo-player">
-        </iframe>
-      </div>
-      <p class="video-coordinates">{{ videoItem.description }}</p>
-      <p class="video-coordinates">Latitude: {{ videoItem.gps_latitude }}</p>
-      <p class="video-coordinates">Longitude: {{ videoItem.gps_longitude }}</p>
-    </div>
-    <ng-template #noVideo>
-      <p class="no-video-message">{{ message }}</p>
-    </ng-template>
-  </ng-template>
-</div>
+  <p class="video-coordinates">{{ videoItem.description }}</p>
+  <p class="video-coordinates">Latitude: {{ videoItem.gps_latitude }}</p>
```

Comment:
Should use camel case for properties in Angular. Or did you do this intentionally to keep your properties named the same as in your backend?

```suggestion
  <p class="video-coordinates">Latitude: {{ videoItem.gpsLatitude }}</p>
```

Reply:
Yeah just using the same names as Rust and the database